package.json

```diff
@@ -28,5 +28,7 @@
     "lint": "tslint -c tslint.full.json --project tsconfig.json --type-check --exclude 'examples/**/*' --exclude '**/node_modules/**' --exclude '**/*.d.ts' '**/*.ts'",
     "test": "mocha --opts test/mocha.opts.root 'packages/*/test/**/*.ts'",
     "posttest": "npm run lint"
+  },
```

Comment:
This file shouldn't be touched at all. `git checkout origin/master -- package.json && git commit --ammend` should do the trick.